packages/table/src/locator.ts

```diff
@@ -122,8 +156,22 @@ export class Locator implements ILocator {
         return bodyRect.left + this.grid.getCumulativeWidthAt(index);
     }
 
+    private convertCellMidpointToClientX = (index: number) => {
+        const bodyRect = this.getBodyRect();
+        const cumWidth = this.grid.getCumulativeWidthAt(index);
+        const prevCumWidth = (index > 0) ? this.grid.getCumulativeWidthAt(index - 1) : 0;
+        return bodyRect.left + ((cumWidth + prevCumWidth) / 2.0);
```

Comment:
`/ 2` === `/ 2.0` in JS. it's always floating point. i'd say drop the decimal cuz we don't do that elsewhere.